charts/traction/values.yaml

```diff
@@ -138,10 +116,11 @@ acapy:
     auto-write-transactions: true
     emit-new-didcomm-mime-type: true
     emit-new-didcomm-prefix: true
+    endorser-alias: endorser
     endorser-protocol-role: author
-    # endorser-alias: endorser
-    # endorser-public-did: 'Ket75eV5UQvVkW2XBjgDH7'
-    # genesis-url: http://test.bcovrin.vonx.io/genesis
+    endorser-public-did: 'Ket75eV5UQvVkW2XBjgDH7'
+    genesis-transactions-list: /home/aries/ledgers.yml
+    genesis-url: http://test.bcovrin.vonx.io/genesis
```

Comment:
`endorser-public-did` and `genesis-url` should NOT be here now that we use the new multi-ledger settings in `ledgers.yml`